tests/build/build.go

```diff
@@ -2,6 +2,7 @@ package build
 
 import (
 	"context"
+	"encoding/json"
```

Comment:
Since these new tests will replace the tests supporting old annotations (when deprecated) .. I think it would be better to keep these new annotation test changes in a separate file like 'build_with_new_annotation.go` which will replace the `build.go` in the future. So that not to mix things together. WDYT?